guides/posts/AI-apis/post.md

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
The name of the file should be "ai-apis/post.md". The directory name should be in lowercase as well. Please do the same for the other file as well.

Reply:
Done!